hydra-node/src/Hydra/Chain/Direct/Tx.hs

```diff
@@ -799,6 +805,7 @@ data ContestObservation = ContestObservation
   { contestedThreadOutput :: (TxIn, TxOut CtxUTxO, ScriptData)
   , headId :: HeadId
   , snapshotNumber :: SnapshotNumber
+  , closedContesters :: [Plutus.PubKeyHash]
```

Comment:
Should: change naming of this. It is confusing because a contest transaction will have contesters before and after. The prefix "closed" here would indicate it's from the close tx (starting) list of contesters. In fact, it's the new, observed list of `contesters`. Just naming it like that would be less confusing.